src/EndOfYearBanner/index.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from 'react';
+import { ThemeProvider } from '@emotion/react';
+import {
+    Button,
+    buttonThemeReaderRevenueBrandAlt,
+    LinkButton,
+    SvgCross,
+    SvgInfo,
+} from '@guardian/source-react-components';
+import { OphanComponentEvent } from '@guardian/libs';
+
+import { BrazeClickHandler } from '../utils/tracking';
+import { styles as commonStyles } from '../styles/bannerCommon';
+import { styles } from './styles';
+import { canRender, COMPONENT_NAME } from './canRender';
+export { COMPONENT_NAME };
+
+export type BrazeMessageProps = {
+    header?: string;
+    body?: string;
+    boldText?: string;
+    buttonText?: string;
+};
+import type { ButtonTheme } from '@guardian/source-react-components';
+
+export type Props = {
+    logButtonClickWithBraze: BrazeClickHandler;
+    submitComponentEvent: (componentEvent: OphanComponentEvent) => void;
+    ophanComponentId?: string;
+    brazeMessageProps: BrazeMessageProps;
+};
+
+const catchAndLogErrors = (description: string, fn: () => void): void => {
+    try {
+        fn();
+    } catch (e) {
+        console.log(`Error (${description}): `, e.message);
+    }
+};
+
+const urlObject = new URL(
+    'https://www.theguardian.com/info/ng-interactive/2020/dec/21/the-guardian-in-2020',
```

Comment:
I guess let's not merge this until we've got the correct destination sorted!?

Reply:
Actually, should this just be a prop that marketing supply? That's what we do for epics - they provide the full URL with any tracking params through Braze.

Reply:
Agreed. Changed this so that buttonUrl can and must be passed in.